Automattic/wp-calypso
# Gutenboarding: rollback to v5

#### Changes proposed in this Pull Request

_Copied from #44980_

- [x] Add domains back as a mandatory step after site name input
- [x] Hide header domain nudge while at domains step
  - In V5 domain nudge is hidden while plans grid is open, but I don't see issue just leaving it there.
- [x] Hide vertical input for everyone (perhaps hide behind a feature flag for now?)
  - [x] Remove test definition
- [x] Update the a/b tests slugs too (both the gutenboarding main one and the existing users test)

#### Testing instructions

1. Go to `/new/`, you shouldn't see vertical input. 
2. Fill your site name, the button text should be "Choose a domain", not "Choose a design".
3. In Domain step, the domain nudge shouldn't show.
4. Open the plans modal by clicking "View Plans", the domain button on the top left should remain visible.


Fixes #44980
